### ISSUE TITLE
Autocomplete suggestion list partially hidden in array format table

### ISSUE THREAD

**Issue by simontaurus** [json-editor/json-editor#1512]
### General information

- json-editor version: 2.14.1

### Expected behavior

Autocomplete suggestion list displayed correctly (works for format != table)
![grafik](https://github.com/json-editor/json-editor/assets/52674635/7b4a0728-62fc-4e38-ae35-bd96244c0647)

### Actual behavior

Autocomplete suggestion list partially hidden
![grafik](https://github.com/json-editor/json-editor/assets/52674635/ff4ffa3c-11cc-4f0c-b812-53d6457ca3ad)

Workaround: 
```css
/* fix autocomplete result lists are not visible in table layouts */
.je-ready .table-responsive {
    overflow-x: unset;
}
```

### Steps to reproduce the behavior

Direct link to example: [playground](https://pmk65.github.io/jedemov2/dist/demo.html?schema=EQbwOgdgBFbALgTwA4FM4C5bAPYCMArVAY3jgBpIY54BLeAG3WCzgEEBXeHYnAW2RN4qKKgAeAQwFMKVbMgBOONArqoAzpijhoMbAH0AFqgkATVAq069emimatg6%2BAtoQA5rN024AMxwKfBJkLNiGtKbmEF421Aj0MqHsXDz8gqjCopLSzJTetsDm6sSuyHQ40UnAADzIAHwAKuHqWVLpUOqGOADuLQBuEq44HC3qGRzILSNu7lDwxlASKbw5maim9AHkUHgSY6ZQFXML1RJQhgqovgC8YHfAhvDwkxgA9K9L3CvpwgB0Lqg%2BgFUIhLr8Vq8AMSvAgSAbFUrwAC03wqqAgZHuc0G7gyt3u%2BjwDAkEAA1nc4HVPqlVqhqh86lAGLQ8ApBohflAmiIxvAJlB%2FApRBJiIYmTgcKT%2Bej3G4RLQWuZfHKDm5Fh00MRaBIGFBiDqGLtiKSmQr4L96fUYrE4MpyhBNKFrLE4u5XKZ9LwGBw%2BA6tABGABMchsAF8Q1BQ3kfAhxCEsM6Y0g0Fo4IM2YhrTHBUF49h4BIibkI3F6Kg%2BI6EyWCsmHNhnK4PFmbcAc8FU8Bqd8hMX8kmEnW4AB1Wik2hoDZnBgSqXIKAACgAChJcXMB%2BoAJTNmNFErj%2B0dgCiGIs6uEYngh18UD4blofB9UAAzHrDIMRcIFOpftuCnbaBUlbaNWMZuhEno4N6vpAQAbNGLoFF2aQ9lYIEtmMgyih2GEKKK%2BjdKO47rNqv4unAuLwAAShoHAMPAABqOocIOwAUdR6i0QxTGoPhhETiRwDwQhBSXBA5gKOxnEdqJ4mSXRvFjvxEikS21IAMqoEwpBaC4zFoTA4Z9nohkuiZxlyIZobAEAA&value=ETI%3D&code=EQehAIEsDsGMBsCuATApgYQMqYBQB1gALAF2IAcBnALjFmWgCsKA6BAexQDN4BDAJ1Ss2AWxA8GPAB4h4kAEYUQ3HgHMAtJFhtoa2BUUBmZkYAMIPYuXrN21voIBKPNGfOw4AOLw2cnvHCwfvC%2BsADW4LIUxM4A7jDIbDHMAFKYAPIAcgCiyJDEbHzMaJw8iPDELIHwwTxhFOAAvOAA3s7g4ASl%2BVrCZPCoxKgEVC1t7eDuACqEkPWzAUEh4ZyIcMSQ2vWcBeDEhKgdwF1sPX0DQ8DgqLn5fGPt7gCS0Lv74BSw%2B8I84ACeHOABJxUHxdmxXgcqjUwuAVmsNi85L9wKFUL97hMIBk2IMRgBGKLgMj8HjCc6gmALapLKD1ACCABkAOp0gCamHAPyBINQcAO%2BQhAUQfAE0GIVxuBWYGPcmHB%2F0Q4GgqGuYM5yGQnPAADd%2BJAeHJ%2Bmq9pDFrVwgLCGx4Jq9vMcLJURCZRACExrnkCgB9D23Ag6vUGo2Uk1XSSks71OSobxJJwuF7jdwMthsUKIMjgRAUGAqcDIHjEH6cPgiWEFYQYggUVD8T5e7xVYaw1awdbad61vifHDuyV8H39gA0UGgZEQxAco0T4wDoNVTT7noHvqlTG0q8KKgGOWXOAA5KWccwyGx9PJ%2BgAxbQVfcOADcGPakE44BwMHHxGY%2FWgKj24AAHnAPFwAAH1A8AAEJrinVoZ1nQEBmFF4AG0AF1H3g9oAF8nznfNCx%2BJprmYbdiAANT8RBUBwB88IEYhkIIotmE4SByhBHA4TbBF30GYRYLw9oGKYvJUGEZh8hTGIQXQHga1o5gon4ComTyQh3zHCdJLYaTZPkmiHDorDwGw4zZ2wocq3g2UkMzbZQQAETpFkIlTdMyCrYAazrQgvQLGIeGbbj2xeHzuw0pdbkHZcRw%2FCdBJM3VQWFfwmn3EhyGoMAAp4ZgeBiFhkFCEBtVQBhoB4bVlTECcThEM5BgAfgARwafdwAAaiuOA2DQABVAAlR5NM%2FYz6KQvgXmVGJwAABVLYRZhokLeIEChrTKxKEPGF833ir8fz%2FQhAPAAxtp28YRKmxCNvgMqcHQ8zLtwhNLthAYe1ShxJP2aAuNbUKcHW09oBrC73uul4Qc2QR13%2B56drM37eQB%2BEOxwAsiwhy71s2misZ4RGELMzCkeJyyvLIwbUAoMoKKo1B%2FIKoLgBGVaO2p2n6copAaKi71NxHdb6Zx4TJuh7nykk1BQiiMnxkpt6kwgTA7PLUE1NCSAyA9H5vDTDMvPC%2Bs4m13XclZ9nAYRTtfN7Gt%2Bxi244q0ydpx25Ksz4NLwAy0hKBoEBeWYM2db15gChUEAYjEMhIBPQgyCa2pQoaFrqL4X4ADJImIBoTcIbOHO%2BfP4ezgpIBUGAGgAKmzig%2BEL9qup6rQBuG0aEswibGJumb5sW5a0Z4jG8fu1Axd218u8O3ljtO86PfexC%2B8lu6HqehWkdcEz2mBYgvp9n6TX%2BjmXmB2nQfB5fIYl27r7hjaEe3kmT7%2BkfQrfQmp9nceHsJswDOIJfhKS7J8YmFlIGmQplZeCBAuZ03KLzaiXow4W31MFG2nMBg0yQQzPmDtNzOwKMLKW7s4I7Shrdemkk8j9FfkrWcBBRRoD4Hg%2BmaDIDmz1lg9G%2F0BYrmHDQ8oI4yClkoFPahKF9wAVkH7Vu4i2CUFbvuAAfPuOB71ZG5G1AEXg%2BgGgEDDmodYxB%2BgEA0a3EW0szFGm6rIkAuiNFaMujoyAeiEDyQoEY4AJiKDQB1rraIwA1EAQoN8aoVjuo2K%2FAEoJAxVHhMifAMJTiPEuKEn7ACMhIAaLQswBgbAYAHjvIwuBLpwBq0YvZHY61D4cDFHwSAtNmCoEVFoJpLT6gGw8sbcBfl6laFWMQZptM%2BGjzCgMohTshajjGrfcYXtUqND9plQOYAhmNNGd0tpiAQD1NKgAJhAJVMkIAOrdV5O3VAQ0RoHXGiZahA8FoiGHufN8%2F9J6LNnHtWe3557%2FiAkvShK9qFfMemhaBOFd7vQPkfeA79UYfMvhQR%2Bv8rr3xhmDJ%2B2haKvygSjM%2B2CL4%2Fx%2BQhP5kE6Qih4KA2Y1K%2BC0sxoRIyzEiLgHQgShCEKf5cpwtA0mGImHjAQbg8hKCmZbJGWMigEyv6IJ5ozGZy4SF8DIfgqRWLyHMDOagcpXlWEgg4eUL0UqunjLZi2fhyropzNiWIiRFBNVrw5bI%2BRlyiSOtUZkve2TdH6O8b44Z5qKCmPoRcH12jwnEheF4wxBArAaE6bCXg1hOlqA9bE%2FK8AyCEB4Ec9AfVBBSUSPpBSU4HGWJyWing0A1EKJidq3VST0nakjW4gC%2Fq40%2BIIMGnZtM1DxLIMEqtESghqLkvHIs8ARiZu1YEKdfgkljqiTk5xmismONkPkwpxT%2Fr7jKUKsYr1sI92gFMN42xqiJFzBEGAtNOQCHAN8aAWMCjIlrZqAQPAEjQHgKA8ArIASBBeD%2BzUsBsz5GEAEItnI5BsDKpyV94BoyxmlIEmegjRzKT5GwV8cRX2JBSOkbI%2FYpyCKKLTUZbBfj4ucFhpoA8CMJCSKkTIu5biYxOIgMkYogGZ1%2BGrfobYCj4GAAAYnhmoTcagS6OBHAwa4LL7zACAA%3D%3D&style=ETI%3D&theme=bootstrap4&iconlib=fontawesome5&object_layout=grid&template=default&show_errors=change&required_by_default=0&no_additional_properties=0&display_required_only=0&remove_empty_properties=0&keep_oneof_values=1&ajax=0&ajaxCredentials=0&show_opt_in=0&disable_edit_json=0&disable_collapse=0&disable_properties=0&disable_array_add=0&disable_array_reorder=0&disable_array_delete=0&enable_array_copy=0&array_controls_top=0&disable_array_delete_all_rows=0&disable_array_delete_last_row=0&prompt_before_delete=1&lib_aceeditor=0&lib_autocomplete=1&lib_sceditor=0&lib_simplemde=0&lib_select2=0&lib_selectize=0&lib_choices=0&lib_flatpickr=0&lib_signaturepad=0&lib_mathjs=0&lib_cleavejs=0&lib_jodit=0&lib_jquery=0&lib_dompurify=1)

```javascript
{
  "type": "object",
  "title": "Autocomplete example",
  "properties": {
    "text": {
      "type": "array",
      "format": "table",
      "items": {
        "type": "string",
        "format": "autocomplete",
        "title": "Wikipedia lookup (Page titles)",
        "description": "Enter a text of minimum 3 characters.",
        "options": {
          "grid_columns": 6,
          "autocomplete": {
            "search": "search_wikipedia",
            "getResultValue": "getResultValue_wikipedia",
            "renderResult": "renderResult_wikipedia",
            "autoSelect": true
          }
        }
      }
    }
  }
}
```

**Comment by germanbisurgi** [comment]
Removing overflow-x from table responsive will break the tables:

![table-broken](https://github.com/json-editor/json-editor/assets/13135260/7e9a0eab-5538-44b5-aac4-f246fd3b377a)

By just changing the position of the autocomplete-result-list it will be displayed correctly inside tables. Conviniently, the autocomplete-result-list has a max-height rule that prevents the editor occupying too much space.

![fixed-table-autocomplete](https://github.com/json-editor/json-editor/assets/13135260/1a682df2-410a-4365-bfea-d7220dc257ba)

I already made a PR for this #1551

**Comment by germanbisurgi** [comment]
fixed in #1551
